src/index.js

```diff
@@ -0,0 +1,8 @@
+import {subwayApp} from './subwayApp.js'
+import {stationInitialize, lineInitialize} from './init.js'
```

Comment:
`Air BNB` 컨벤션 코드에 따르면 `import` 문에 대한 컨벤션은 공백이 들어가더라구용!

```suggestion
import { subwayApp } from './subwayApp.js'
import { stationInitialize, lineInitialize } from './init.js'
```

정말 별거 아니지만.. 중괄호 시작하고랑, 끝나기 전이랑 공백을 추가해주는 것으로 해보자구욥!